kuma/search/models.py

```diff
@@ -128,6 +127,8 @@ def promote(self):
             index_objects.delay(mapping_type, [instance.id])
             rescheduled.append(label)
         self.outdated_objects.all().delete()
+        # Allow only a single index to be promoted.
+        Index.objects.all().update(promoted=False)
```

Comment:
Currently it still does the filter and orders by `-created`. I could swap the order so it updates the `self` promoted value, then removes promoted from all excluding `self`?